includes/functions.php

```diff
@@ -34,12 +34,46 @@
  *
  * @param array $args Arguments for fetching stories.
  *
- * @return string|void
+ * @return void
  */
 function stories( $args = [] ) {
-	$story_query = new Story_Query( $args );
-	//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
-	echo $story_query->render();
+	$attrs = [];
+
+	if ( empty( $args ) ) {
+		//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo get_plugin_instance()->customizer->render_stories();
```

Comment:
Why is this getting a render method from a customizer class. Seems a little misnamed / confusing.

Reply:
Customizer has few defaults set for stories, this function will render the stories based on the Customizer settings if arguments have not been passed explicitly.

Reply:
From a developer experience perspective that sounds a bit unexpected.

Perhaps two functions make more sense?

* `render_stories( array $args = [] )` to display stories, with some sensible defaults (or just do `render_stories( array $args )`
* `render_stories_from_customizer()` to just do `echo get_plugin_instance()->customizer->render_stories();`

The second one would need a better name I think.

Reply:
The normal pattern in WP is, have one function get to the string version and one to get the string version and echo it. 

Think of [get_the_title](https://developer.wordpress.org/reference/functions/get_the_title/) and [the_title](https://developer.wordpress.org/reference/functions/the_title/).

Can we follow this pattern here?

Reply:
> Can we follow this pattern here?

That sounds like a different issue?

To be honest, the WP pattern of using functions like `get_the_title` and `the_title` is not really the best one. I don't think we need to follow core to a tee here.

Sure, we can have a `get_stories()` function, but that doesn't solve the case where the default values are coming from the customizer, which I find unexpected. Also, what would `get_stories()` even return? Just an array (`$story_query->get_stories()`)?

We could do something like this:

* `get_stories( array $args )` - returns an array of stories, easy enough
* `render_stories( array $args )` - render stories with some sensible defaults
* `render_theme_stories( array $args )` - render customizer stories

Reply:
> get_stories( array $args ) - returns an array of stories, easy enough
render_stories( array $args ) - render stories with some sensible defaults
render_theme_stories( array $args ) - render customizer stories

Works for me